backend/routers/transcribe.py

```diff
@@ -65,7 +67,7 @@ def get(request: Request):
 
 @router.websocket("/listen")
 async def websocket_endpoint(
-        websocket: WebSocket, uid: str, language: str = 'en', sample_rate: int = 8000, codec: str = 'pcm8',
+        websocket: WebSocket, uid: str = Depends(auth.get_current_user_uid),language: str = 'en', sample_rate: int = 8000, codec: str = 'pcm8',
```

Comment:
Does this work when the websocket opens?

Reply:
Yes! I first tested the changes with this endpoint only

Reply:
Alright then please comment it for now.

And I will add as /v1/listen so that existing users can keep using the current endpoint when we deploy the backend